### PR TITLE
Create Rule “team-works-in-vertical-slices/rule”

### PR DESCRIPTION
**Pull request by JackDevAU**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/team-works-in-vertical-slices/rule.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+type: rule
+title: Does your team work in vertical slices?
+uri: team-works-in-vertical-slices
+authors:
+  - title: Adam Cogan
+  - title: Jack Pettit
```
Comment (bradystroud) [inline rules/team-works-in-vertical-slices/rule.md:7]:
```suggestion
authors:
  - title: Adam Cogan
    url: https://ssw.com.au/people/adam-cogan
  - title: Jack Pettit
    url: https://ssw.com.au/people/jack-petit
```

File: rules/team-works-in-vertical-slices/rule.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+type: rule
+title: Does your team work in vertical slices?
+uri: team-works-in-vertical-slices
```
Comment (bradystroud) [inline rules/team-works-in-vertical-slices/rule.md:4]:
```suggestion
uri: work-in-vertical-slices
```
If you agree, also update the folder name

File: rules/team-works-in-vertical-slices/rule.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Part of Scrum means all developers are responsible for the work that is conducted during a sprint. To push this point, as part of our definition of done; Adding 2 other developers to look over the code allows for,
+
+1. The quality of code to remain high
+2. Still allows for all developers to learn (Even those reviewing) 
+3. Teaching the author of the PR 
+4. Asking questions
+
+
+![Figure: Vertical slicing is just like a slice of cake!](vertical-slicing-new-post-img-min.webp)
```
Comment (bradystroud) [inline rules/team-works-in-vertical-slices/rule.md:29]: Stick to png or jpg. Gatsby does fancy image perf optimisation stuff and prefers png or jpg.

File: rules/team-works-in-vertical-slices/rule.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  - title: Adam Cogan
+  - title: Jack Pettit
+related:
+  - the-team-do-you-encourage-multi-skilled-teams-by-leaving-your-comfort-zone
+created: 2023-02-22T04:02:12.393Z
+guid: d85ead8c-9a6b-4715-b798-e3db535604f8
+---
+As a Software Developer you often work with both the front and backend. Most people like to work with which ever is their strong suit. 
+
+That doesn't help get your weaker skills stronger. Lets look at another way to increase your weaker skills.
```
Comment (bradystroud) [inline rules/team-works-in-vertical-slices/rule.md:15]:
```suggestion
That doesn't help get your weak skills stronger. Lets look at another way to strengthen your weaker skills.
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: rules/team-works-in-vertical-slices/rule.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+<!--endintro-->
+
+When working in teams it is often normal to split the work into what peoples strong suits are. This can be beneficial but it comes with a flaw of not helping developers strengthen their weaker skills. At SSW we prefer to do a whole slice of work each time (from the front end to the API, and any extras in between). 
+
+Part of Scrum means all developers are responsible for the work that is conducted during a sprint. To push this point, as part of our definition of done; Adding 2 other developers to look over the code allows for,
+
+1. The quality of code to remain high
+2. Still allows for all developers to learn (Even those reviewing) 
+3. Teaching the author of the PR 
+4. Asking questions
```
Comment (Hona) [inline rules/team-works-in-vertical-slices/rule.md:26]: I'd add a point to this that while a vertical slice is great for someone to get across the project - you need to spread the IP knowledge across the team, as losing someone becomes an increased risk

**Comment by tiagov8** [comment]
Hey @JackDevAU 

1. Please add this rule to a category 

PS. You should use the same PR. Ping me on Teams if unsure

FYI - I have made a bunch of updates to improve the rule reading. The most important one is that we don't use "At SSW, we..." on Rules. Rules should be generic for anyone to follow ;)

**Comment by JackDevAU** [comment]
>     1. Please add this rule to a category

Done - Add to rules to better Scrum

**Review by tiagov8 — APPROVED** [review]

**Review by bradystroud — APPROVED** [review]

**Pull request merged**